src/WebJobs.Script/Config/FeatureFlags.cs

```diff
@@ -12,18 +14,52 @@ namespace Microsoft.Azure.WebJobs.Script.Config
     /// </summary>
     public static class FeatureFlags
     {
+        private static readonly object _cacheLock = new();
```

Comment:
For private static, do we use `s_` prefix? Should we use `s_` to differentiate from instance?